user/puzzles/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "puzzles"
+pkgver = "20240802"
+pkgrel = 0
+build_style = "cmake"
```

Comment:
```suggestion
build_style = "cmake"
configure_args = ["-DNAME_PREFIX=puzzles-"]
```

otherwise they all have super common basic names (like `cube`), this neatly namespaces everything